src/components/Filter.res

```diff
@@ -291,9 +291,81 @@ let make = (
   <Form onSubmit initialValues=initialValueJson>
     <AutoSubmitter autoApply submit=onSubmit defaultFilterKeys />
     {<AddDataAttributes attributes=[("data-filter", "remoteFilters")]>
-      <div>
-        <div className={`flex gap-3 items-center flex-wrap ${verticalGap}`}>
-          {customLeftView}
+      {<>
```

Comment:
Can we break this component into small chunks? It's getting difficult to read the code in this manner.

Like value computation outside JSX and assigning values to small small components

Reply:
moved the allFilters UI outside main component